apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/matcher/TracePathMatcher.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.skywalking.apm.plugin.trace.ignore.matcher;
+
+public interface TracePathMatcher {
```

Comment:
I can't find this matcher used in any place. What is the purpose of this interface?

Reply:
I think there may be a variety of path matching rules. This interface is designed to support this extension